coalib/output/ConsoleInteraction.py

```diff
@@ -855,38 +855,58 @@ def show_enumeration(console_printer,
     console_printer.print()
 
 
+def check_deprecation(show_description,
+                      show_params):
+    """
+    Shows a deprecation warning message if the parameters
+    show_description or show_params are not None.
+
+    :param show_description: This parameter is being checked
+                             for deprecation.
+    :param show_params:      This parameter is being checked
+                             for deprecation.
+    """
+    param_dict = {'show_description': show_description,
+                  'show_params': show_params}
+    for param_name, param_value in param_dict.items():
+        if param_value is not None:
+            logging.warning('{} parameter is deprecated'.format(param_name))
+
+
 def show_bear(bear,
-              show_description,
-              show_params,
               console_printer,
-              args=None):
+              args=None,
+              show_description=None,
```

Comment:
Please do not change the order of arguments.  That is a breaking change.  Stuff breaks (in other repositories) whenever someone does that, and the person who breaks it doesnt fix the other repositories.